scss/_patterns_image.scss

```diff
@@ -1,6 +1,41 @@
 @import 'settings';
 
 @mixin vf-p-image {
+  %vf-image-container--common {
+    .p-image-container__image {
+      object-fit: contain;
+    }
+    &.is-highlighted {
+      background: $colors--theme--background-alt;
+    }
+    align-items: center;
+    display: grid;
+    justify-content: center;
+    text-align: center;
+  }
+
+  .p-image-container,
+  [class^='p-image-container--'] {
+    @extend %vf-image-container--common;
```

Comment:
Is this placeholder only extended here? Maybe the styles can be defined here directly, as there doesn't seem to be a need for the abstraction yet.

Reply:
Yes, I abstracted it early so it would be easier to extend later; but I've gone ahead and moved its styling to this selector